src/display/editor/annotation_editor_layer.js

```diff
@@ -484,6 +484,11 @@ class AnnotationEditorLayer {
     return editor;
   }
 
+  addNewEditor() {
+    const { width, height } = this.div.getBoundingClientRect();
+    this.#createAndAddNewEditor({ offsetX: width / 2, offsetY: height / 2 });
+  }
+
```

Comment:
This can be removed, since we've already added such a method in the meantime; see https://github.com/mozilla/pdf.js/blob/c377f2d48860ce7c9aba9dbf1f6f4835762c7d77/src/display/editor/annotation_editor_layer.js#L553-L560